enterprise/frontend/src/metabase-enterprise/content_translation/use-translate-content.unit.spec.tsx

```diff
@@ -0,0 +1,148 @@
+import { setupEnterprisePlugins } from "__support__/enterprise";
+import { mockSettings } from "__support__/settings";
+import { renderHookWithProviders } from "__support__/ui";
+import { useListContentTranslationsQuery } from "metabase-enterprise/api";
+import type { TokenFeatures } from "metabase-types/api";
+import type { RetrievedDictionaryArrayRow } from "metabase-types/api/content-translation";
+import {
+  createMockTokenFeatures,
+  createMockUser,
+} from "metabase-types/api/mocks";
+import { createMockState } from "metabase-types/store/mocks";
+
+import { useTranslateContent } from "./use-translate-content";
+
+jest.mock("metabase-enterprise/api", () => ({
+  useListContentTranslationsQuery: jest.fn(),
+}));
+
+const mockUseListContentTranslationsQuery =
+  useListContentTranslationsQuery as jest.Mock;
+
+interface SetupOpts {
+  locale: string | undefined;
+  /** An untranslated string */
+  msgid: string | null | undefined;
+  translations: RetrievedDictionaryArrayRow[] | undefined;
+  tokenFeatures?: Partial<TokenFeatures>;
+  hasEnterprisePlugins?: boolean;
+}
+
+const sampleSpanishDictionary: RetrievedDictionaryArrayRow[] = [
+  {
+    id: 1,
+    locale: "es",
+    msgid: "Hello World",
+    msgstr: "Hola Mundo",
+  },
+];
+
+function setup({
+  locale,
+  msgid,
+  translations,
+  tokenFeatures = {},
+  hasEnterprisePlugins = true,
+}: SetupOpts) {
+  if (hasEnterprisePlugins) {
+    setupEnterprisePlugins();
+  }
```

Comment:
you shouldn't need this since you're already importing the enterprise hook directly